py-polars/polars/series/series.py

```diff
@@ -3289,8 +3289,14 @@ def has_validity(self) -> bool:
         """
         Return True if the Series has a validity bitmask.
 
-        If there is none, it means that there are no null values.
-        Use this to swiftly assert a Series does not have null values.
+        If there is no mask, it means that there are no ``null`` values.
+
+        Notes
+        -----
+        While the _absence_ of a validity bitmask guarantees that a Series does not
+        have ``null`` values, the converse is not true, eg: the _presence_ of a
+        bitmask does not mean that there _are_ null values, as every value of the
+        bitmask could be ``false``.
```

Comment:
Maybe a reference to the `null_count()` or are you avoiding cross reference?

Reply:
No, that's a good idea actually; will update in a sec.

Reply:
Done: https://github.com/pola-rs/polars/pull/11323.